pylint/lint.py

```diff
@@ -674,7 +674,10 @@ def process_tokens(self, tokens):
                             self.add_message('file-ignored', line=start[0])
                             self._ignore_file = True
                             return
-                        meth(msgid, 'module', start[0])
+                        sharp_list = content.split('#')
```

Comment:
I am not entirely sure I understand this, can you simplify the condition by splitting it into multiple clauses and assign variables names to each? Also, would this disable the following lines as well, if I understand correctly from your comments? I don't think we should do that, if that is the case.